### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Gaurav0**
I think we should release 5.0.0 as soon as possible.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "broccoli-lint-eslint",
-  "version": "4.2.1",
+  "version": "5.0.0",
```
Comment (Turbo87) [inline package.json]: this should be done using `yarn version` to make sure that the commit is tagged correctly

Comment (Gaurav0) [reply package.json]: Done.

**Comment by Turbo87** [comment]
did you test this out inside of ember-cli-eslint yet?

**Comment by Gaurav0** [comment]
@Turbo87 I have tested with my work app, it does in fact produce lots of new eslint errors. :)

**Comment by Gaurav0** [comment]
@Turbo87 Anything I can do to help get a release out?

**Comment by Gaurav0** [comment]
@Turbo87 @rwjblue bump? Can we release this?

**Pull request merged**